### DISCUSSION TITLE
File manager Strawberry uses?

### DISCUSSION THREAD

**Discussion by mbrandis** [strawberrymusicplayer/strawberry#1063] [Q&A] [answered]
Hi!

I'm wondering based on which configuration setting Strawberry is starting the file manager from the right click menu "show in file browser". When I use this, Visual Studio Code opens, which definitely is not the file manager I want to use. ;-)

My system is Debian testing/unstable with XFCE and I searched in ${home}/.config for this but couldn't find it. My guess is, that this is a qt configuration, but I couldn't find anything.

Thanks for any help!

Cheers!

**Comment by jonaski** [comment, marked as answer]
It uses: `xdg-mime query default inode/directory` to find the desktop file for what is configured under Default Applications / utilities / File Manager.

**Reply by mathias-ch1903** [reply]
Thanks for your answer! Had the same issue, found your hint, now it's working again (Debian/xfce4)!
